mne/io/artemis123/artemis123.py

```diff
@@ -0,0 +1,274 @@
+
+    # Channel Names by type
+    ref_mag_names = ['REF_001', 'REF_002', 'REF_003',
+                     'REF_004', 'REF_005', 'REF_006']
+
+    ref_grad_names = ['REF_007', 'REF_008', 'REF_009',
+                      'REF_010', 'REF_011', 'REF_012']
+
+    # load mne loc dictionary
+    loc_dict = _load_mne_locs()
+    info['chs'] = []
+    info['bads'] = []
+
+    for i, chan in enumerate(header_info['channels']):
+        # build chs struct
+        t = {'cal': float(chan['scaling']), 'ch_name': chan['name'],
+             'logno': i + 1, 'scanno': i + 1, 'range': 1.0,
+             'unit_mul': FIFF.FIFF_UNITM_NONE,
+             'coord_frame': FIFF.FIFFV_COORD_DEVICE}
+        t['loc'] = loc_dict.get(chan['name'], np.zeros(12))
+
+        if (chan['name'].startswith('MEG')):
+            t['coil_type'] = FIFF.FIFFV_COIL_ARTEMIS123_GRAD
+            t['kind'] = FIFF.FIFFV_MEG_CH
+            # While gradiometer units are T/m, use T here as values are more
+            # similar to what you get with neuromag magnetometers in terms of
+            # scale due to long baseline.
+            # This choice also matches the one used for CTF
+            t['unit'] = FIFF.FIFF_UNIT_T
+            t['unit_mul'] = FIFF.FIFF_UNITM_F
+
+        # 3 axis referance magnetometers
+        elif (chan['name'] in ref_mag_names):
+            t['coil_type'] = FIFF.FIFFV_COIL_ARTEMIS123_REF_MAG
+            t['kind'] = FIFF.FIFFV_REF_MEG_CH
+            t['unit'] = FIFF.FIFF_UNIT_T
+            t['unit_mul'] = FIFF.FIFF_UNITM_F
+
+        # reference gradiometers
+        elif (chan['name'] in ref_grad_names):
+            t['coil_type'] = FIFF.FIFFV_COIL_ARTEMIS123_REF_GRAD
+            t['kind'] = FIFF.FIFFV_REF_MEG_CH
+            # While gradiometer units are T/m, use T here as values are more
+            # similar to what you get with neuromag magnetometers in terms of
+            # scale due to long baseline.
+            # This choice also matches the one used for CTF
+            t['unit'] = FIFF.FIFF_UNIT_T
+            t['unit_mul'] = FIFF.FIFF_UNITM_F
+
+        # other reference channels are unplugged and should be ignored.
+        elif (chan['name'].startswith('REF')):
+            t['coil_type'] = FIFF.FIFFV_COIL_NONE
+            t['kind'] = FIFF.FIFFV_MISC_CH
+            t['unit'] = FIFF.FIFF_UNIT_V
+            info['bads'].append(t['ch_name'])
+
+        elif (chan['name'].startswith('AUX')):
+            t['coil_type'] = FIFF.FIFFV_COIL_NONE
+            t['kind'] = FIFF.FIFFV_MISC_CH
+            t['unit'] = FIFF.FIFF_UNIT_V
+
+        elif (chan['name'].startswith('TRG')):
+            t['coil_type'] = FIFF.FIFFV_COIL_NONE
+            t['kind'] = FIFF.FIFFV_STIM_CH
+            t['unit'] = FIFF.FIFF_UNIT_V
+
+        elif (chan['name'].startswith('MIO')):
+            t['coil_type'] = FIFF.FIFFV_COIL_NONE
+            t['kind'] = FIFF.FIFFV_MISC_CH
+            t['unit'] = FIFF.FIFF_UNIT_V
```

Comment:
These lines could be condensed to ``if chan['name'].startswith(('AUX', 'TRG', 'MIO')):``.